src/RO/index.ts

```diff
@@ -30,18 +32,26 @@ const data: SpectrumData[] = [
     names: ["B20"],
     spectrumData: Band20,
   },
+  {
+    names: ["B28", "n28"],
+    spectrumData: Band28,
+  },
   {
     names: ["B32"],
     spectrumData: Band32,
   },
   {
-    names: ["B38"],
+    names: ["B38", "n41"],
```

Comment:
Is the NR actually deployed as n41 or n38? The range is inside n38 so I would have assumed it'd be deployed as n38.